### DISCUSSION TITLE
求高手帮忙

### DISCUSSION THREAD

**Discussion by RiseToTop** [ant-design/ant-design-pro#11068] [Q&A]
{
name:"xxx",
path:"xxx",
component: './tailor'
}
这是我动态获取的菜单数据，菜单正常显示，但一点击就404,是哪里写的不对吗，用的是antd pro和umijs4

**Comment by Fettes** [comment]
你菜单点击怎么写的。
检查你的menu组件，看看路由怎么调转的，有没有拿到路由
